src/scripts/main.js

```diff
@@ -1,3 +1,41 @@
 'use strict';
 
-// write code here
+const thead = document.querySelector('thead');
+const tbody = document.querySelector('tbody');
+const tbodyElements = Array.from(tbody.querySelectorAll('tr'));
+
+const sorting = (item) => {
+  const sortedRows = tbodyElements.sort((a, b) => {
+    switch (item) {
+      case 'Name':
+        return a.firstElementChild.textContent.localeCompare(
+          b.firstElementChild.textContent,
+        );
+      case 'Position':
+        return a.children[1].textContent.localeCompare(
+          b.children[1].textContent,
+        );
+      case 'Age':
+        return a.children[2].textContent - b.children[2].textContent;
+      case 'Salary':
+        const cellA = a.children[3].textContent
+          .replace('$', '')
+          .replace(',', '');
+        const cellB = b.children[3].textContent
+          .replace('$', '')
+          .replace(',', '');
+
+        return cellA - cellB;
```

Comment:
Ensure that `cellA` and `cellB` are parsed as numbers before performing subtraction. Use `parseFloat` or `Number` to convert the string values to numbers after removing the currency symbol and commas.